process/sync/basicForkDetector.go

```diff
@@ -113,12 +113,6 @@ func (bfd *basicForkDetector) checkBlockValidity(header data.HeaderHandler, stat
 	if int64(header.GetRound()) > bfd.rounder.Index() {
 		return ErrHigherRoundInBlock
 	}
-	if header.GetNonce() == bfd.fork.checkpointNonce {
```

Comment:
I think this should be remove by the other PR.

Reply:
Done